bin/build-release-ccd-definition.sh

```diff
@@ -15,9 +15,9 @@ elif [ ${environment} == demo ]; then
 elif [ ${environment} == local ]; then
   excludedFilenamePatterns="-e *-prod.json"
 elif [ ${excludeNonProdFiles} == true ]; then
-  excludedFilenamePatterns="-e UserProfile.json,*-nonprod.json,*-COS-nonprod.json,*GAspec.json,*-GAR2GAspec-nonprod.json,*DJ.json,*DJspec.json,*DJ-SDO-nonprod.json,*DJ-nonprod.json,*-HNL-nonprod.json,*CUI.json"
+  excludedFilenamePatterns="-e UserProfile.json,*-nonprod.json,*-COS-nonprod.json,*GAspec.json,*-GAR2GAspec-nonprod.json,*DJ.json,*DJspec.json,*DJ-SDO-nonprod.json,*DJ-nonprod.json,*-HNL-nonprod.json,*CUI.json,*-GAR3CP-nonprod.json"
```

Comment:
do we need to explicitly specify to exclude "*-GAR3CP-nonprod.json"? 

*-nonprod.json will exclude "*-GAR3CP-nonprod.json" too

Reply:
yes i've been told to do so during pr meeting